Lib/test/test_builtin.py

```diff
@@ -772,6 +772,48 @@ def test_exec_redirected(self):
         finally:
             sys.stdout = savestdout
 
+    def test_exec_closure(self):
+        result = 0
+        def make_closure_functions():
+            a = 2
+            b = 3
+            c = 5
+            def two_freevars():
+                nonlocal result
+                nonlocal a
+                nonlocal b
+                result = a*b
+            def three_freevars():
+                nonlocal result
+                nonlocal a
+                nonlocal b
+                nonlocal c
+                result = a*b*c
+            return two_freevars, three_freevars
+        two_freevars, three_freevars = make_closure_functions()
+
+        exec(two_freevars.__code__,
+            two_freevars.__globals__,
+            closure=two_freevars.__closure__)
+        self.assertEquals(result, 6)
```

Comment:
```suggestion
        self.assertEqual(result, 6)
```
assertEquals is deprecated

Reply:
Done.